platform/broadcom/saibcm-modules/debian/opennsl-modules-4.9.0-7-amd64.init

```diff
@@ -54,7 +54,9 @@ start)
 stop)
     echo -n "Unload OpenNSL kernel modules... "
```

Comment:
To avoid confusion, his message should be removed if we comment out the later function.

Reply:
correct, only one is needed. :)